lib/assessment.md

```diff
@@ -41,7 +50,10 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+.big {}
+#contentPane {}
+.important .red {}
```

Comment:
We were looking for the intersection, but I would also have accepted the union given that the wording of the prompt is a little ambiguous. Unfortunately, the answer you gave was the third.